index.html

```diff
@@ -735,93 +920,138 @@ <h2 class="subscribe-block__title title">
             alt=""
           />
 
-          <div class="footer-section__our-contacts">
-            <a class="footer-section__logo logo link-black" href="#">
-              J<span class="decor-orange">o</span>bply
+          <div class="footer__our-contacts">
+            <a class="logo link-black" href="#">
+              J<span class="decor_orange">O</span>BPLY
             </a>
-            <p class="footer-section__address">
-              152/3 , Western Road, New York
-            </p>
-            <p class="footer-section__call-us hover-orange">
+            <p class="footer__address">152/3 , Western Road, New York</p>
+            <a href="tel:+99-8569-5234" class="footer__call-us hover_orange">
               Call Us: +99-8569-5234
-            </p>
+            </a>
 
-            <div class="footer-section__social-media">
+            <div class="footer__social-media">
               <a class="social-media__icons" href="#">
```

Comment:
```suggestion
              <a class="social-icon" href="#">
```
`social-media__icons` - check my previous comment on creating elements of elements. It's semantically incorrect.